src/components/ImportFromGoogleDriveDialog/index.js

```diff
@@ -74,20 +74,16 @@ export default ({ open, onClose, onAddSamples }) => {
 
   const createPicker = useCallback(() => {
     if (pickerApiLoaded && oauthToken) {
-      const view = new window.google.picker.View(
-        window.google.picker.ViewId.Docs
-      )
-
-      view.setMimeTypes(
-        "image/png",
-        "image/jpeg",
-        "image/jpg",
-        "image/gif",
-        "video/mp4",
-        "video/mpeg"
-      )
+      const view = new window.google.picker.DocsView(
+        window.google.picker.ViewId.Docs,
+        window.google.picker.ViewId.FOLDERS
+      ).
+      setIncludeFolders(true).
+      setSelectFolderEnabled(true);
```

Comment:
to fix the tests you just need to run `npm run prettier` in the root directory, that'll automatically correct the code styling :)